pkg/modulereader/resreader.go

```diff
@@ -39,16 +41,74 @@ type VarInfo struct {
 	Required    bool
 }
 
+// OutputInfo stores information about module output values
+type OutputInfo struct {
+	Name        string
+	Description string `yaml:",omitempty"`
+	Sensitive   bool   `yaml:",omitempty"`
+	// DependsOn   []string `yaml:"depends_on,omitempty"`
+}
+
+// UnmarshalYAML supports parsing YAML OutputInfo fields as a simple list of
+// strings or as a list of maps directly into OutputInfo struct
+func (mo *OutputInfo) UnmarshalYAML(value *yaml.Node) error {
+	var name string
+	const yamlErrorMsg string = "block beginning at line %d: %s"
+
+	err := value.Decode(&name)
+	if err == nil {
+		mo.Name = name
+		return nil
+	}
+
+	var fields map[string]interface{}
+	err = value.Decode(&fields)
+	if err != nil {
+		return fmt.Errorf(yamlErrorMsg, value.Line, "outputs must each be a string or a map{name: string, description: string, sensitive: bool}")
```

Comment:
Here you discarding info from `err`, this may be useful to include it.

Reply:
Added a `log` entry. As an example of replacing the map syntax with a list (that should fail):

```
  line 52: cannot unmarshal !!seq into map[string]interface {}
2023/04/10 12:11:17 failed to parse the blueprint in simple-vm.yaml, check YAML syntax for errors, err=block beginning at line 52: outputs must each be a string or a map{name: string, description: string, sensitive: bool}
```